### DISCUSSION TITLE
Laravel version

### DISCUSSION THREAD

**Discussion by mbnoimi** [microweber/microweber#930] [Q&A] [answered]
README.md says [microweber](https://github.com/microweber/microweber) uses Laravel 8 while [composer.json](https://github.com/microweber/microweber/blob/master/composer.json#L51) says it uses Laravel 9.1.*

Which version of Laravel microweber use?

**Comment by peter-mw** [comment, marked as answer]
Hi, we are using Laravel 9

**Reply by mbnoimi** [reply]
Thanks.
